crates/ruff_notebook/src/notebook.rs

```diff
@@ -205,6 +205,22 @@ impl Notebook {
         })
     }
 
+    /// Creates an empty notebook.
+    ///
+    ///
+    pub fn empty() -> Self {
+        Self::from_raw_notebook(
+            RawNotebook {
+                cells: vec![],
+                metadata: RawNotebookMetadata::default(),
+                nbformat: 4,
+                nbformat_minor: 5,
+            },
+            false,
+        )
+        .unwrap()
+    }
```

Comment:
Technically, an empty notebook would contain at least a single empty cell. I verified that by creating an empty notebook in VS Code and Jupyter Lab. The use-case for this seems to be only when there's some error when creating a Ruff notebook in Red Knot. I think it make sense to use this representation (one empty code cell) for an empty notebook.